site/docs/v1/tech/messengers/generic-messenger.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: doc
+title: Generic Messenger
+description: Adding and Configuring Messengers - Generic
+---
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Set Up a Generic Messenger
+
+Currently the only supported transport for a messenger is SMS. If you need to connect to an SMS provider other than Twilio, you can use the Generic messenger configuration.
```

Comment:
I'm thinking perhaps we add a little note to answer the question, "what is a transport?".  We introduce this concept with MFA but don't really define it as SMS is the only supported protocol.

When configuring SMS for MFA on the Tenant or Application, the transport protocol/type (of which is currently hard-coded/read-only) tells us which Messengers are compatible for the SMS configuration.  

In the future, the idea is that we have other transport protocols (Ie. Email) such that you can pair up your messaging protocols on your Tenant or Application with your messenger.

So maybe something like:
```
The transport value of `SMS` tells FusionAuth that a messenger can be assigned to a link:/docs/v1/tech/guides/multi-factor-authentication/#tenant-set-up[Set Up a Generic Messenger][multi-factor configuration] on an Application or Tenant.
```

Reply:
🤔 Or perhaps maybe we add this sort of explanation to the Overview?  @mooreds thoughts?

Reply:
After discussion

> Currently the only supported transport for a messenger is SMS
Currently, messengers can only be configured to send messages via a SMS protocol.